kratos/python/add_utilities_to_python.cpp

```diff
@@ -104,80 +104,73 @@ void AddUtilitiesToPython(pybind11::module& m)
-        .def("SetToZero_VectorVar", &VariableUtils::SetToZero_VectorVar)
-        .def("SetToZero_ScalarVar", &VariableUtils::SetToZero_ScalarVar)
-        // .def("SetToZero_VelocityVectorVar", &VariableUtils::SetToZero_VelocityVectorVar)
-        // .def("CheckVariableExists", &VariableUtils::CheckVariableExists< Variable<double> >)
-        // .def("CheckVariableExists", &VariableUtils::CheckVariableExists< VariableComponent< VectorComponentAdaptor<array_1d<double, 3> > > > )
-        // .def("CheckVariableExists", &VariableUtils::CheckVariableExists< Variable<array_1d<double, 3> > > )
-        .def("ApplyFixity", &VariableUtils::ApplyFixity<Variable<double>>)
-        .def("ApplyFixity", &VariableUtils::ApplyFixity<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("ApplyVector", &VariableUtils::ApplyVector<Variable<double>>)
-        .def("ApplyVector", &VariableUtils::ApplyVector<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("SumHistoricalNodeScalarVariable", &VariableUtils::SumHistoricalNodeScalarVariable<Variable<double>>)
-        .def("SumHistoricalNodeScalarVariable", &VariableUtils::SumHistoricalNodeScalarVariable<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("SumHistoricalNodeVectorVariable", &VariableUtils::SumHistoricalNodeVectorVariable)
-        .def("SumNonHistoricalNodeScalarVariable", &VariableUtils::SumNonHistoricalNodeScalarVariable<Variable<double>>)
-        .def("SumNonHistoricalNodeScalarVariable", &VariableUtils::SumNonHistoricalNodeScalarVariable<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("SumNonHistoricalNodeVectorVariable", &VariableUtils::SumNonHistoricalNodeVectorVariable)
-        .def("SumConditionScalarVariable", &VariableUtils::SumConditionScalarVariable<Variable<double>>)
-        .def("SumConditionScalarVariable", &VariableUtils::SumConditionScalarVariable<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("SumConditionVectorVariable", &VariableUtils::SumConditionVectorVariable)
-        .def("SumElementScalarVariable", &VariableUtils::SumElementScalarVariable<Variable<double>>)
-        .def("SumElementScalarVariable", &VariableUtils::SumElementScalarVariable<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("SumElementVectorVariable", &VariableUtils::SumElementVectorVariable)
-        .def("AddDof", &VariableUtils::AddDof<Variable<double>>)
-        .def("AddDof", &VariableUtils::AddDof<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("AddDof", &VariableUtils::AddDofWithReaction<Variable<double>>)
-        .def("AddDof", &VariableUtils::AddDofWithReaction<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("CheckVariableKeys", &VariableUtils::CheckVariableKeys)
-        .def("CheckDofs", &VariableUtils::CheckDofs);
+    class_<VariableUtils > (m,"VariableUtils")
```

Comment:
Do you really need to change all these lines?

Reply:
It is just a tabulation correction